oddbird/slide-deck
# Add slide parts (and mark private methods)

![](https://source.unsplash.com/featured/?cute,animal&CHANGE_ME)



## Related Issue(s)
_Reminder to add related issue(s), if available._

- #14 - at least a partial fix?
- #4 - with support for hiding either (but not both) the notes and the slide

## Steps to test/reproduce

There's only one sample slide with notes near the end of the demo. But the control panel (command-k) has buttons for hiding one or the other.

I think this branch raises a few questions:
- I called the three parts a `slide-group`, `slide-frame`, and `slide-note`. Can we come up with better names? `slide-container` and `slide-canvas`? Should it be `slide-notes` plural?
- We can't provide shadow-dom default styles for anything nested inside a slotted element. Should we find a way around that, or stop providing default slide styles at all, or…?
- Should this be documented as the _primary_ use-case, or is it good that we allow a simpler all-children-are-slides setup? That may be a moot point if we remove the default styles.
- If we do remove the shadow-dom styles, that maybe makes it more important that we provide a 'theme' people can apply.
